### ISSUE TITLE
[BUG] Ignore paths usage

### ISSUE THREAD

**Issue by loncarales** [stackrox/kube-linter#1165]
**System info:**
 - OS: Ubuntu 24.04.3 LTS

**Describe the bug**

I am not really sure if this is an actual bug or a feature. I am trying to run kube-linter to lint Flux manifests for my clusters.  I would like to exclude everything under `flux-system` subfolder (`gotk-components.yaml` ...), but I can't seem to make it work 😃 

**To Reproduce**

```bash
$ kube-linter version
0.8.3

$kube-linter lint ./clusters
KubeLinter 0.8.3
No lint errors found!

$ kube-linter lint ./clusters --add-all-built-in
KubeLinter 0.8.3
...
/somet-path/clusters/stg/flux-system/gotk-components.yaml: (object: flux-system/image-automation-controller apps/v1, Kind=Deployment) object has a restart policy defined with '' but the only accepted restart policies are '[Always OnFailure]' (check: restart-policy, remediation: Set up the restart policy for your object to 'Always' or 'OnFailure' to increase the fault tolerance.)

Error: found 180 lint errors

$ kube-linter lint ./clusters --add-all-built-in --ignore-paths "/**/*/flux-system/**/*"
KubeLinter 0.8.3

Error: found 180 lint errors
```

Tried different combinations of ignore-paths regex, without any success.

 `kube-linter lint ./clusters --ignore-paths "./clusters/**/*/flux-system/**/*" --add-all-built-in`

**Expected behavior**

Linter should ignore all files under the `flux-system` subfolder.

**Comment by loncarales** [comment]
Figure it out 😺
